test/sa_db_users.sql

```diff
@@ -76,3 +76,6 @@ GRANT SELECT ON registrations TO 'badkeyrevoker'@'localhost';
 
 -- Test setup and teardown
 GRANT ALL PRIVILEGES ON * to 'test_setup'@'localhost';
+
+-- Test only privilege, not reflective of prod
+GRANT ALTER on fqdnSets TO 'sa'@'localhost';
```

Comment:
What test requires this privilege? I know we'd talked about adding this to a test but all I see here are deletes.

Reply:
Thank you, forgot to delete this. I was using the ALTER privilege to try to reset the AUTO_INCREMENT counter on the fqdnSets table to force the insert to fail, and then forgot to remove this after I gave up and removed that test instead.